test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -6199,7 +6216,7 @@ class GRUNoSeqLengthWithoutStateModel(torch.nn.Module):
             def __init__(self, layers, bidirect, dropout, batch_first):
                 super(GRUNoSeqLengthWithoutStateModel, self).__init__()
                 self.batch_first = batch_first
-                self.inner_model = torch.nn.GRU(RNN_INPUT_SIZE, RNN_HIDDEN_SIZE, num_layers=layers, 
+                self.inner_model = torch.nn.GRU(RNN_INPUT_SIZE, RNN_HIDDEN_SIZE, num_layers=layers,
```

Comment:
nit: Several of these lines are not related to the change but just space removals.